src/week_number.jsx

```diff
@@ -1,16 +1,31 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import classnames from 'classnames'
 
 export default class WeekNumber extends React.Component {
   static propTypes = {
-    weekNumber: PropTypes.number.isRequired
+    weekNumber: PropTypes.number.isRequired,
+    onClick: PropTypes.func
+  }
+
+  handleClick = (event) => {
+    if (this.props.onClick) {
+      this.props.onClick(event)
+    }
   }
 
   render () {
+    let weekNumberClasses = {
```

Comment:
prefer const to let. Even though we are changing a property of weekNumberClasses, the reference to the object does not change so it doesn't need to be let